### DISCUSSION TITLE
rec模型 resnet34 和 resnet18之类，哪种层数的准确率高

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12616] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：win10
- 版本号/Version：Paddle：  PaddleOCR：2.6 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

rec模型 resnet34 和 resnet18之类，哪种层数的准确率高

**Comment by QQQTAO** [comment]
> 请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
> 
> * 系统环境/System Environment：win10
> * 版本号/Version：Paddle：  PaddleOCR：2.6 问题相关组件/Related components：
> * 运行指令/Command Code：
> * 完整报错/Complete Error Message：
> 
> 我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):
> 
> rec模型 resnet34 和 resnet18之类，哪种层数的准确率高

模型越大肯定识别越准确

**Comment by nissansz** [comment]
灰度图和彩图，用于训练，哪种识别准确率高？
